tests/e2e/scripts/ibmcloud_settings.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+cat > $TMP_DIR/oadpcreds <<EOF
+{
+  "spec": {
+      "configuration":{
+        "velero":{
+          "defaultPlugins": [
+            "openshift", "$PROVIDER"
+          ]
+        }
+      },
+      "backupLocations": [
+        {
+          "velero": {
+            "provider": "$PROVIDER",
+            "config": {
+              "profile": "$BSL_AWS_PROFILE",
+              "region": "$BSL_REGION",
+              "s3ForcePathStyle": "true",
```

Comment:
@Mathieu-Ferraton @kaovilai  Wont this set `s3ForcePathStyle` for all the SPECs that are running ? I think we explicitly set these in some tests and remove in some tests.

Reply:
@deepakraj1997 You're right, but I think these are aws specific tests, which we want to skip for ibmcloud clusters (this can be discussed further though). If we use IBM COS bucket as BSL with ibmcloud/powerVS clusters, then we need to keep this to `true`, or fork an ibmcloud plugin out of the aws one.

Reply:
@deepakraj1997 This is fine. It would apply to all specs within the ibmcloud which have to use aws plugin via the s3ForcePathStyle to work.